conf/application.conf

```diff
@@ -129,6 +129,7 @@ mongodb {
 urls {
   login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:9758/disclose-cross-border-arrangements"
+  homepage = "http://localhost:9758/disclose-cross-border-arrangements"
```

Comment:
Same as loginContinue. No need to duplicate

Reply:
@daniel-manning

Reply:
separation of concerns, loginContinue is the gg process and this is just what our start/home page is